evasquez26/angular4-startup
# S2-ONE-131 cancel role request

## Story / Task

see [ONE-131](https://bixlabs.atlassian.net/browse/ONE-131)


## What this PR does?

Adds functionality to cancel a role request once the user has made one already


### Before 
Could not cancel a role request

### After

The user is able to cancel a role request

### How to test / reproduce

- login to the app
- go to profile -> roles
- if you hvae crated role requests, you'll see the option to cancel
- after clicking cancel the request should be removed


## Migrations
NO


Thanks!
